include/dblayer/mysqli.php

```diff
@@ -188,7 +188,7 @@ function close()
 	{
 		if ($this->link_id)
 		{
-			if ($this->query_result)
+			if (is_object($this->query_result))
```

Comment:
MySQLi returns an instance of the `mysqli_result` class, so this would be cleaner:

```php
if ($this->query_result instanceof mysqli_result) {
```